docs/_includes/generated/failure_rate_restart_strategy_configuration.html

```diff
@@ -10,14 +10,14 @@
     <tbody>
         <tr>
             <td><h5>restart-strategy.failure-rate.delay</h5></td>
-            <td style="word-wrap: break-word;">"1 s"</td>
-            <td>String</td>
+            <td style="word-wrap: break-word;">1000ms</td>
+            <td>Duration</td>
             <td>Delay between two consecutive restart attempts if <span markdown="span">`restart-strategy`</span> has been set to <span markdown="span">`failure-rate`</span>. It can be specified using notation: "1 min", "20 s"</td>
         </tr>
         <tr>
             <td><h5>restart-strategy.failure-rate.failure-rate-interval</h5></td>
-            <td style="word-wrap: break-word;">"1 min"</td>
-            <td>String</td>
+            <td style="word-wrap: break-word;">60000ms</td>
```

Comment:
as a future note for the generator, if the value in milliseconds is a lot larger we may want to display this in seconds/minutes instead.